src/main/java/org/mifos/connector/channel/camel/config/CamelProperties.java

```diff
@@ -10,5 +10,6 @@ private CamelProperties() {}
     public static final String CLIENTCORRELATIONID = "X-CorrelationID";
     public static final String BATCH_ID_HEADER = "X-BatchID";
     public static final String PLATFORM_TENANT_ID = "Platform-TenantId";
+    public static final String PAYMENT_SCHEME_HEADER = "X-PaymentScheme";
```

Comment:
API Guidelines indicate that new headers have to follow hyphen separated naming convention like X-Payment-Scheme.